### PR TITLE
chore: update to fix security vulnerability in fast-viewer-react

### PR DESCRIPTION
**Pull request by awentzel**
# Description
* Update webpack related packages to newer versions
* Update to optimize webpack using inline-source-maps in production

## Motivation & context
This solves the security vulnerability of using Webpack < 3.1.11.  https://nvd.nist.gov/vuln/detail/CVE-2018-14732

## Issue type checklist
- [x] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

## Process & policy checklist

- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://microsoft.github.io/fast-dna/docs/en/contributing/standards) for this project.

### CODE REVIEW

File: packages/fast-viewer-react/webpack.config.js [hunk 1]
```diff
@@ -1,50 +1,54 @@
...
-                    }
-                ],
-                exclude: /node_modules/
-            }
-        ]
-    },
-    plugins: [
-        new HtmlWebpackPlugin({
-            title: "FAST viewer",
-            contentBase: outDir
```
Comment (nicholasrice) [inline packages/fast-viewer-react/webpack.config.js]: Does all of this really pertain to a security vulnerability? This seems out of scope

File: packages/fast-viewer-react/webpack.config.js [hunk 2]
```diff
@@ -1,50 +1,54 @@
...
+
+    const isProduction = args.mode === "production";
+
+    return {
+        devServer: {
+            compress: false,
+            historyApiFallback: true,
+            open: true,
+            port: 3000
+        },
```
Comment (janechu) [inline packages/fast-viewer-react/webpack.config.js]: Why are we moving all the keys around and changing this to be a function?

Comment (awentzel) [reply packages/fast-viewer-react/webpack.config.js]: I followed the optimizations put in place for inline-source-maps by Nick.  I did alphabetize the properties to cleanup and make it easier to find things while I was in there.   I mentioned in the task description the changes I made.

File: packages/fast-viewer-react/webpack.config.js [hunk 3]
```diff
@@ -1,50 +1,54 @@
...
-        rules: [
-            {
-                test: /\.tsx?$/,
-                use: [
-                    {
-                        loader: "ts-loader",
-                        options: {
-                            compilerOptions: {
-                                declaration: false,
+module.exports = (env, args) => {
```
Comment (janechu) [inline packages/fast-viewer-react/webpack.config.js]: please remove these changes as they are not related to the security vulnerability indicated by the PR title

File: packages/fast-viewer-react/webpack.config.js (hunk 2 above)
Comment (janechu) [reply packages/fast-viewer-react/webpack.config.js]: please remove these changes as they are not related to the security vulnerability indicated by the PR title which is the only thing that will be recorded in git history, we can have a separate PR that updates all the webpack configs, I think that would make the most sense.

**Pull request closed without merging**